betterconf/caster.py

```diff
@@ -56,6 +57,43 @@ def cast(self, val: str) -> typing.Union[int, typing.NoReturn]:
             raise ImpossibleToCastError(val, self)
 
 
+class FloatCaster(AbstractCaster):
+    def cast(self, val: str) -> typing.Union[float, typing.NoReturn]:
+        val = val.replace(",", ".")
+        try:
+            as_float = float(val)
+            return as_float
+        except ValueError:
+            raise ImpossibleToCastError(val, self)
+
+
+class ListCaster(AbstractCaster):
+    separator: str = ","
```

Comment:
I think we could probably add a constructor that accepts the separator:
```python
def __init__(self, separator = ","):
    super().__init__(ListCaster, self)  # this is the syntax I remember, might've changed
    self.separator = separator
```

Reply:
I don't see why we need `super().__init__(ListCaster, self)` line